# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/bglibs/no-selftests.patch
# Copyright (C) 2006 - 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- bglibs-2.04/Makefile.vanilla	2022-03-31 22:54:58.620717316 +0200
+++ bglibs-2.04/Makefile	2022-03-31 22:55:02.541717519 +0200
@@ -69,7 +69,7 @@
 adt/hashs.lo adt/hashs.o: ltcompile adt/hashs.c include/bglibs/adt_common.h
 	./ltcompile adt/hashs.c
 
-all: sysdeps.h libraries programs man selftests
+all: sysdeps.h libraries programs man
 
 base64/asc2bin.lo base64/asc2bin.o: ltcompile base64/asc2bin.c include/bglibs/base64.h include/bglibs/str.h sysdeps.h
 	./ltcompile base64/asc2bin.c
